MacOS: Errors building fsdev.c

```
In file included from /Users/travis/build/redhatrises/openscap/src/OVAL/probes/fsdev.c:53:
/usr/include/sys/ucred.h:91:2: error: unknown type name 'u_long'; did you mean 'long'?
        u_long  cr_ref;                 /* reference count */
        ^
/usr/include/sys/ucred.h:133:9: error: unknown type name 'u_int'
        u_int   cr_version;             /* structure layout version */
        ^
In file included from /Users/travis/build/redhatrises/openscap/src/OVAL/probes/fsdev.c:54:
In file included from /usr/include/sys/mount.h:76:
/usr/include/sys/attr.h:77:2: error: unknown type name 'u_short'; did you mean 'short'?
        u_short bitmapcount;                    /* number of attr. bit sets in list (should be 5) */
        ^
/usr/include/sys/attr.h:537:2: error: unknown type name 'u_long'; did you mean 'long'?
        u_long                          maxmatches;
        ^
/usr/include/sys/attr.h:550:2: error: unknown type name 'u_char'; did you mean 'char'?
        u_char                          ss_fsstate[548];        // fs private
        ^
5 errors generated.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -381,6 +381,10 @@ if(WIN32)
 	# eg. inet_pton
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_WIN32_WINNT=0x0600")
 endif()
+if(APPLE)
+	#full Single Unix Standard v3 (SUSv3) conformance (the Unix API)
+	add_definitions(-D_DARWIN_C_SOURCE)
+endif()
 
 include_directories(
 	"compat/"

